src/connection.ts

```diff
@@ -42,8 +44,13 @@ export const configureConnection = (option: ConfigureOption = {}) => {
   }
 }
 
+export const setConnectionOptions = (options: TypeORMConnectionOptions): void => {
```

Comment:
Everything working well in my project from this PR locally, thanks again. One thing is I think this should probably be like `setConnectionOptions = (options: Partial<TypeORMConnectionOptions>)` so you can do something like `setConnectionOptions({ logging: true })` without needing `setConnectionOptions({ logging: true } as any)`

Reply:
thx for the tip
add it to #59